module/purpose/ScriptRunner/README.md

```diff
@@ -0,0 +1,22 @@
+# Script Runner
+
+## Description
+
+Allows scripts to be run from the command line.
+
+## Usage
+
+ScriptRunner will attempt to run each command line parameter as a script. Once
+the first script has completed it will run the next, and so on until all are
+finished at which point it terminates the program.
+
+## Consumes
+
+- `CommandLineArguments` containing the list of scripts from the command line
+- `message::AllServoWaypointsComplete` indicates that a script has finished
```

Comment:
That's no used here, plus even if such a thing were it should be handled by a Director Done event